src/tools/thmviewer/load.cpp

```diff
@@ -118,6 +130,15 @@ static DWORD WINAPI LoadThreadProc(
         }
         else
         {
+            if (sczWxlPath)
+            {
+                hr = LocLoadFromFile(sczWxlPath, &pWixLoc);
+                ExitOnFailure(hr, "Failed to load loc file from path: %ls", sczWxlPath);
+
+                hr = ThemeLocalize(pTheme, pWixLoc);
```

Comment:
Looks like this won't handle reloads on the `.wxl` file the way `.thm` files are handled. Maybe something for the future.